### DISCUSSION TITLE
Sir PAL1000... I have a question (Realtek Driver MSI)

### DISCUSSION THREAD

**Discussion by Klipperken** [msys2/MINGW-packages#7832] [Q&A]
Greetings everyone:

Should this not be the proper way to contact sir PAL1000, then I sincerely apologize.
I looked for a way to write him, but there seems to be no option.

Please do ask sir Pal1000 to respond though.

Thank you very much...

Greetings, good sir PAL1000:

I have an MSI MAG B460 Tomahawk.
Installing a driver simply was not possible, the computer kept reverting to the Windows base driver.
FINALLY found your driver, which installed flawlessly.
Sadly though, it does not contain a Sound Manager, and the Windows Sound controller is... not good.

I want to ask you if you know a way to install a real Realtek Sound Manager upon your Realtek Driver, or that you could provide a way do do so.

If it cannot be done... then there's a great pathos here...
A monumental sorrow...
But i will still be equally grateful, for the Driver alone already, that you provided...

Thank you, sir PAL1000.

**Comment by Biswa96** [comment]
Why did you create a discussion here? You can create an issue in the repository that you have used. msys2 does not deal with any system driver.

**Reply by Klipperken** [reply]
AH, my apologies.
I am totally new here, I am not familiar with the workings of this site yet.

Could you please elaborate/explain what you mean, sir?
I am not native English speaking, and I did not quite get what you were saying there...

Thank you.

**Comment by pal1000** [comment]
For your inforrmation, @Klipperken, you should have started an issue [here](https://github.com/pal1000/Realtek-UAD-generic/issues).

Do you have Realtek Audio Console/Control app? If no you can get it from [Windows store](https://www.microsoft.com/store/productId/9P2B8MCSVPLN). If you don't have Windows store, e.g. removed or LTS, you can do an [offline installation](https://github.com/pal1000/Realtek-UAD-generic/releases). You probably need to [read this](https://www.howtogeek.com/285410/how-to-install-.appx-or-.appxbundle-software-on-windows-10/) to learn how to do it.

**Reply by Klipperken** [reply]
Hi sir.

Before anything, well met.
And sincere apologies.
I was not aware, thank you for the information regarding posting issues.

Now, I DO have the Windows Store Sound Control app, issue is that the quality of said app is far from the one Realtek Sound Manager has.

This is why I asked if it was possible somehow to install the Manager and get rid of the Console one.

My BIG issue is autism.
I am virtually a walking radio, but music has to sound "right" (which, I am fully aware, is a taste matter), and for some reason, I just can't get the console to sound RIGHT.

The sound from Console is "hollow" and "pover", coming from the Realtek Manager, where the sound is "full" and "rich", which gives me the urge to walk outside and kill random folks, rofl.
It is so frustrating, this poor sound, I have not heard a song in like a week and a half because of it.
Actually, the last week I've been in bed due to the miss of music.
I come out of bed to check for mail, to do a few tasks, but the lack of music just drains my energy in such way, I "need" to go to bed...

IF it is possible to revert to Manager from Console, I would be so happy.
If not, I am... screwed...

I will then have to rebuild an old system, and have that as my system, I fear...

Sir, thank you so much for responding and trying to help me, I truly appreciate that!!!

Kindest regards:
Ben

**Reply by pal1000** [reply]
It looks like you need a legacy HDA driver. Post on one of these forums your sound device hardware ID:
- https://forums.mydigitallife.net/threads/update-realtek-high-definition-audio.72236/
- https://station-drivers.com/index.php?option=com_kunena&view=topic&catid=18&id=17&Itemid=858&lang=en
- https://www.tenforums.com/sound-audio/

**Reply by Klipperken** [reply]
Wow, thank you SO much, sir.
You have no idea how grateful I am...

Thank you...
Ben
